content/en/docs/tasks/administer-cluster/enabling-topology-aware-hints.md

```diff
@@ -0,0 +1,40 @@
+---
+reviewers:
+- robscott
+title: Enabling Topology Aware Hints
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in EndpointSlices. This approach tries to keep traffic close to where
+it originated from. This can result in decreased costs and improved performance.
+
+## {{% heading "prerequisites" %}}
+
+  {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+The following prerequisites are needed in order to enable topology aware hints:
+
+   * Kubernetes 1.21 or later
+   * {{< glossary_tooltip text="Kube-proxy" term_id="kube-proxy" >}} running in
+     iptables mode or IPVS mode
+   * [Enable Endpoint
+     Slices](/docs/tasks/administer-cluster/enabling-endpointslices)
```

Comment:
nit; I'd write:
```suggestion
   * Make sure that you have [enabled EndpointSlices](/docs/tasks/administer-cluster/enabling-endpointslices/)
```

Reply:
This is a bit of a strange thing to communicate since they're already on by default. I've switched this up to read "Ensure that you have not disabled EndpointSlices", does that make sense?

Reply:
We could actually omit `/docs/tasks/administer-cluster/enabling-endpointslices/` from v1.21 I think: I don't think many cluster operators are turning this API off.

Reply:
Also, this isn't as key a feature as EndpointSlice so if you want to, it's OK to omit `content/en/docs/tasks/administer-cluster/enabling-topology-aware-hints.md` from the PR entirely.

Reply:
Good call, removed this reference entirely. I'm not sure what the bar is for a page like this, but I think it could be a feature that users are interested in testing out early. Happy to remove it if that's preferred.